crates/rethnet_evm/benches/util.rs

```diff
@@ -1,24 +1,54 @@
-use std::clone::Clone;
+use std::{clone::Clone, sync::Arc};
 
 use criterion::{BatchSize, BenchmarkId, Criterion};
+use parking_lot::Mutex;
+use tokio::runtime::Builder;
+
 use rethnet_eth::{Address, Bytes, U256};
-use rethnet_evm::state::{HybridState, LayeredState, RethnetLayer, StateError, SyncState};
+use rethnet_evm::{
+    state::{ForkState, HybridState, LayeredState, RethnetLayer, StateError, SyncState},
+    HashMap, RandomHashGenerator,
+};
 use revm::primitives::{AccountInfo, Bytecode, KECCAK_EMPTY};
 
-#[derive(Default)]
-struct RethnetStates {
+pub struct RethnetStates {
     layered: LayeredState<RethnetLayer>,
     hybrid: HybridState<RethnetLayer>,
+    pub fork: ForkState,
```

Comment:
Benchmarks should support the `test-disable-remote` flag, such that people can run tests if they don't have an alchemy url

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/3937/commits/127e79feb6b5e35eb949a3aa4a7c1189e2fc5299